unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/TextureHelpers/TextureHelpers.cs

```diff
@@ -54,7 +66,10 @@ public static Texture2D Resize(Texture2D source, int newWidth, int newHeight)
     public static Texture2D CopyTexture(Texture2D sourceTexture)
     {
         Texture2D texture = new Texture2D(sourceTexture.width, sourceTexture.height, sourceTexture.format, false);
-        Graphics.CopyTexture(sourceTexture, texture); // TODO: does this work in WebGL?
+        
+        // TODO: WebGL doesn't support this, should we add the ReadPixels/GetPixels approach as in Resize() ?
+        Graphics.CopyTexture(sourceTexture, texture);
```

Comment:
Should we change this with:

```
        bool supportsGPUTextureCopy = SystemInfo.copyTextureSupport != CopyTextureSupport.None;
        if (supportsGPUTextureCopy)
        {
            Graphics.CopyTexture(rt, nTex);
        }
        else
        {
            nTex.ReadPixels(new Rect(0, 0, newWidth, newHeight), 0, 0);
            nTex.Apply();
        }
```

and in the resize function just use:

```
    Graphics.CopyTexture(rt, nTex);
```

?

Reply:
Yeah that's why I updated the comment on that line, I just talked to Brian about this, I'll apply that change today before merging.

Reply:
done!